lib/internal/test_runner/test.js

```diff
@@ -476,7 +479,7 @@ class Test extends AsyncResource {
     return { ctx, args: [ctx] };
   }
 
-  async [kRunHook](hook, args) {
+  async runHook(hook, args) {
```

Comment:
Why this change? It looks unrelated.

Reply:
see the pr description

Reply:
Can we make it two different PRs instead?

Reply:
I don't think there is any benefit to splitting this into two PRs. The commits are related enough that one PR would depend on the other, and they would still land as they are.